sections/03_categorize.md

```diff
@@ -63,6 +63,29 @@ subsequent training. Similar strategies have been employed for EHR data where
 high-quality labeled examples are also difficult to obtain
 [@doi:10.1101/039800].
 
+Chest X-ray image analysis is another area where recent DL methods showed
+remarkable results [@arXiv:1701.06599 @arXiv:XXX]. Alternative to using fully
+labeled training data, the authors in [@arXiv:XXX] proposed to use weakly
+labeled images for training deep learning models. To generate weak labels for
+x-ray images, Wang et al. [1] applied a series of Natural Language Processing
```

Comment:
I'm guessing this refers to one of your citations. Can you use the [@arXiv: ___] syntax?

Also X-ray/x-ray are inconsistent here.